test/fixtures.rb

```diff
@@ -46,3 +46,20 @@ class ArrayMailer
 
   to ["noreply1@example.com", "noreply2@example.com"]
 end
+
+class WelcomeMailer
+  include Lotus::Mailer
+
+  def greeting
+    "Ahoy"
+  end
+end
+
+class User
+  def initialize(locals)
```

Comment:
what do you think about setting the constructor params with keyword arguments `initialize(name:, username:)` ?